playbooks/docker-machine-functional-devstack-mitaka/run.yaml

```diff
@@ -0,0 +1,59 @@
+- hosts: all
+  become: yes
+  roles:
+    - clone-devstack-gate-to-workspace
+    - create-devstack-local-conf
+    - install-devstack
+    - config-golang
+  tasks:
+    - name: Create Docker machine
+      shell:
+        cmd: |
+          set -ex
+          set -o pipefail
+          git clone https://github.com/sstephenson/bats.git
+          pushd bats
+          ./install.sh /usr/local
+          popd
+          # install Docker
+          apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D || true
+          apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D || true
+          apt-add-repository 'deb http://apt.dockerproject.org/repo ubuntu-trusty main'
+          apt-get update
+          apt-cache policy docker-engine
+          apt-get install -y docker-engine=1.12.6-0~ubuntu-trusty
+          cat /lib/systemd/system/docker.service
+          sed -r -i "s|(ExecStart)=(.+)|\1=\2 --iptables=false|" /lib/systemd/system/docker.service
+          cat /lib/systemd/system/docker.service
+          service docker restart
+          ifconfig -a
```

Comment:
Whether we can do some refactor in order to put install-docker into a role?

Reply:
not very sure, but here it is specific for ubuntu trusty. and also in ubuntu 14.04. we can only use `service xxx restart`, the `systemctl` command not supportted.